File: .claude/plugins/openai-agent-builder/references/project-templates/python-multi-agent/specialists/specialist_1.py
"""{{PROJECT_NAME}} — Specialist agent 1."""
from agents import Agent
from tools import tools

specialist_1 = Agent(
    name="{{SPECIALIST_1_NAME}}",
    instructions="{{SPECIALIST_1_INSTRUCTIONS}}",
    handoff_description="{{SPECIALIST_1_HANDOFF_DESCRIPTION}}",
    tools=tools,
    model="gpt-4.1",
)

File: .claude/plugins/openai-agent-builder/references/project-templates/typescript-multi-agent/src/specialists/specialist1.ts
import { Agent } from '@openai/agents';
import { tools } from '../tools';

export const specialist1 = new Agent({
  name: '{{SPECIALIST_1_NAME}}',
  instructions: '{{SPECIALIST_1_INSTRUCTIONS}}',
  tools,
  model: 'gpt-4.1',
});
